backend/src/meetings/notfication-about-meetings.service.ts

```diff
@@ -133,11 +133,50 @@ export class NotificationAboutMeetingsService {
       meeting,
       creator, 
       null
-    )
+    );
     var html = this.renderMail(parts);
     return this.sendMail(attendeeEmails, parts.title, html).then(_ => meeting);
   }
 
+  async notifyUserAboutBooking(user: UserEntity, meeting: MeetingEntity): Promise<MeetingEntity>{
+    if (!this.configService.get(ConfigTokens.MAIL_ENABLED)) {
+        this.logger.warn("Mail is not enabled!");
+        return Promise.resolve(meeting);
+    }
+    const parts = this.renderParts(
+      meeting.isIdea ? mailConfig.notificationMail.ideaParticipationRejected : mailConfig.notificationMail.meetingParticipationRejected, 
+      meeting, 
+      user, 
+      null
+    );
+    const html = this.renderMail(parts);
+    return this.sendMail([user.email], parts.title, html).then(_ => meeting);
+  }
+
+  async notifyAuthorAboutBooking(isNewAttendee: boolean, meeting: MeetingEntity, attendee: UserEntity): Promise<void>{
+    if (!this.configService.get(ConfigTokens.MAIL_ENABLED)) {
+      this.logger.warn("Mail is not enabled!");
+      return;
+    }
+    let template;
+    if (isNewAttendee){
+      template = meeting.isIdea ? mailConfig.notificationMail.newAttendeeForIdea : mailConfig.notificationMail.newAttendeeForMeeting;
+    } else {
+      template = meeting.isIdea ? mailConfig.notificationMail.canceledAttendeeForIdea : mailConfig.notificationMail.canceledAttendeeForMeeting;
+    }
+    const parts = this.renderParts(
+      template,
+      meeting,
+      attendee,
+      null
+    );
+    const html = this.renderMail(parts);
+    const author = await this.usersService.findOne(attendee.username);
+    if (author){
+      return this.sendMail([author.email], parts.title, html).then(() => {});
+    }
```

Comment:
This could be a code smell, because sometimes it could return not the promised Promise